### ISSUE TITLE
Multiple Areas

### ISSUE THREAD

**Issue by chmadkk** [plaksnor/HASS-JablotronSystem#5]
Im trying to get this to work in my home enviroment. 
is it possible to have more than one area ? I cant seem to find the right implementation

**Comment by Shamshala** [comment]
Yes, it's possible. You have to sniff the communication between F-Link and Jablotron (for example via Wireshark+USBPcap).
Probably depends on the version of your system but for my JA-101K, i had to look for `80 02 0d xx` for dis/arming (for example main zone 90/a0, second zone 91/a1) and `51 22 xx xx xx` for the status (interestingly, @plaksnor uses packet[2:3], my system uses [4:5] instead and [2:3] uses for zone specification). :slightly_smiling_face:

**Comment by deryckhans** [comment]
Hello @Shamshala ,

First off all, thanks a lot for picking this up.  Based on your input, I was able to get the jablotron integration working with my JA-100K.  The only problems is that arming the system only arms the bottom zone.  In understand based on you reply above that somewhere in @plaksnor code we can add something to be able to arm other zones aswell, but could you hint on what exactly to change for this?

Many thanks!
Hans.

**Comment by kukulich** [comment]
@deryckhans Can you try https://github.com/kukulich/home-assistant-jablotron100/ ? It's based on this integration but it's rewritten from scratch. It supports more sections/zones.

**Comment by Shamshala** [comment]
@deryckhans Hi, well, @kukulich was faster. I was about to recommend you to try his integration which is far more promising project now and most importantly, maintained (by programmer). Zones are covered perfectly (so in this case you don't need to edit the code), sensors are a bit harder task because the developer's main unit uses slightly different way of communication with sensors than plaksnor's integration (and unfortunately there wasn't much users interested in Jablotron so far last few months) so i'm sure anybody's input will be welcomed.
